src/java.base/share/classes/java/net/InetAddress.java

```diff
@@ -205,6 +206,24 @@
  * <p>
  * A value of -1 indicates "cache forever".
  * </dd>
+ * <dt><b>networkaddress.cache.stale.ttl</b></dt>
+ * <dd>Indicates the caching policy for stale names. The value is specified as
+ * an integer to indicate the number of seconds the stale names will be kept in
+ * the cache. The name is considered stale if it expired and attempts to refresh
+ * it were unsuccessful. This property is useful if it is preferable to use a
+ * stale name rather than result of unsuccessful name lookup. The default
+ * setting is to cache for an implementation specific period of time.
+ * <p>
+ * If the value of this property is larger than "networkaddress.cache.ttl" then
+ * "networkaddress.cache.ttl" will be used as a refresh interval of the name in
+ * the cache. For example, if this property is set to 1 day and
+ * "networkaddress.cache.ttl" is set to 30 seconds, then the positive response
+ * will be cached for 1 day but an attempt to refresh it will be done every
+ * 30 seconds.
+ * <p>
+ * A value of 0 (zero) means do not use stale names, the negative value is
```

Comment:
Should we be more specific here? As far as I could tell, negative values also disable caching.

Reply:
I guess it is an accurate description since the negative value is ignored by the implementation.